internal/cli/alerts/settings/create.go

```diff
@@ -66,7 +65,20 @@ func CreateBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Creates one alert configuration in the specified project.",
-		Args:  require.NoArgs,
+		Long: `The mongocli atlas alerts settings create command creates
+		one alert configuration in the specified project.`,
+		Example: `  This example uses the mongocli atlas alerts settings create
+		command to create one alert configuration in the specified
+		project. It uses the default profile to access the Atlas project:
+		
+		$ mongocli atlas alert settings create --event JOINED_GROUP --enabled \
+		--notificationType USER --notificationEmailEnabled \
+		--notificationUsername john@example.com \
+		 -o json --projectId 5df90590f10fab5e33de2305`,
```

Comment:
when using (`) for multi line strings new lines and spacing is kept as is, don't use this to "style" how it looks on code but how it looks rendered, this currently renders as
```sh
 ./bin/mongocli om alerts settings create --help
The mongocli atlas alerts settings create command creates
		one alert configuration in the specified project.

Usage:
  mongocli ops-manager alerts settings create [flags]

Examples:
  This example uses the mongocli atlas alerts settings create
		command to create one alert configuration in the specified
		project. It uses the default profile to access the Atlas project:

		$ mongocli atlas alert settings create --event JOINED_GROUP --enabled \
		--notificationType USER --notificationEmailEnabled \
		--notificationUsername john@example.com \
		 -o json --projectId 5df90590f10fab5e33de2305
```

note how new lines are added when no needed

```
This example uses the mongocli atlas alerts settings create
		command to create one alert configuration in the specified
```

should be
```
This example uses the mongocli atlas alerts settings create command to create one alert configuration in the specified
```

and snooty is probably going to complain about the extra spacing at the beginning of each line 

```suggestion
		Long: "The mongocli atlas alerts settings create command creates one alert configuration in the specified project.",
		Example: `This example uses the mongocli atlas alerts settings create command to create one alert configuration in the specified project. It uses the default profile to access the Atlas project:
		
  $ mongocli atlas alert settings create --event JOINED_GROUP --enabled \
		--notificationType USER --notificationEmailEnabled \
		--notificationUsername john@example.com \
		 -o json --projectId 5df90590f10fab5e33de2305`,
```

(you may want to double check the actual "code example" is correct by snooty rules

Reply:
These new lines probably happened because of the automatic line wrap in VSCode that we use for other docs. I think I ran into this Julia and forgot to say to disable it when working on annotations, I'm sorry!

Reply:
Addressed